examples/dataproc-running-notebooks/notebooks/sample_notebook.ipynb

```diff
@@ -7,14 +7,109 @@
    "source": [
     "# Sample Notebook\n",
     "The output of this notebook will be executed via PySpark job then will be stored in the specified notebook path in notebook_args in the Composer DAGs. \n",
+    "\n",
     "This notebook let users \n",
+    "* run a sample Spark Session with a sample CSV file\n",
     "* verify if GCS buckets are properly mounted as a file system and\n",
     "* execute Python files that are stored in mounted GCS buckets by !python and %run commands and\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 39,
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run a sample Spark Session with a sample CSV file\n",
+    "\n",
+    "from pyspark.sql import SparkSession\n",
+    "from pyspark.sql.functions import avg, sum, max\n",
+    "spark = SparkSession \\\n",
+    "    .builder \\\n",
+    "    .appName(\"Spark Session for Bike Sharing Data\") \\\n",
+    "    .getOrCreate()\n",
+    "\n",
+    "path=\"gs://kristin_serverless_pyspark/bikesharingdemand-train.csv\"\n",
```

Comment:
Is this csv too big? If it is small is better to have it under a resources folder and not hardcode a GCS path that may not exist in the future